runtime/util/optinfo.c

```diff
@@ -735,6 +735,18 @@ preloadClassNameAtIndex(U_32* preloadInfoPtr, U_32 index)
 	return NNSRP_PTR_GET(preloadClassesPtr, J9UTF8*);
 }
 #endif /* J9VM_OPT_VALHALLA_VALUE_TYPES */
+#if defined(J9VM_OPT_VALHALLA_FLATTENABLE_VALUE_TYPES)
+U_16
+getImplicitCreationFlags(J9ROMClass *romClass)
+{
+	U_32 *ptr = getSRPPtr(J9ROMCLASS_OPTIONALINFO(romClass), romClass->optionalFlags, J9_ROMCLASS_OPTINFO_IMPLICITCREATION_ATTRIBUTE);
+
+	Assert_VMUtil_true(ptr != NULL);
+	U_32* implicitCreationInfo = SRP_PTR_GET(ptr, U_32*);
```

Comment:
This is C code, please move the declaration of `implicitCreationInfo` to the top of the scope before the assertion (to avoid error from some compilers). i.e.

```
U_32* implicitCreationInfo = NULL;

Assert_VMUtil_true(ptr != NULL);
implicitCreationInfo = ...
```